### ISSUE TITLE
Screen reader test should be ignored on mobile (android)

### ISSUE THREAD

**Issue by sculpt0r** [ckeditor/ckeditor4#5202]
## Type of report

Failing test

## Provide detailed reproduction steps (if any)

1. Run manual tests with tag `4.19.0` that requires screen reader

### Expected result

*What is the expected result of the above steps?*

Tests are ignored.

### Actual result

*What is the actual result of the above steps?*

I need to perform the test.

## Other details

* Browser: Chrome (mobile)
* OS: Android
* CKEditor version: 4.19.0 (pre-release)
* Installed CKEditor plugins: full all

**Comment by jacekbogdanski** [comment]
I would avoid ignoring screen reader tests on mobiles. These tests should be still performed on iOS with VoiceOver. A developer can simply skip tests on android to avoid too strict browser sniffing.
